policies/policy-change-process.md

```diff
@@ -67,4 +69,18 @@ and closing the pull request without merging.
 If the policy change is approved, the pull request is merged to the
 master branch of the general-policies repository.
 
+Exceptions
```

Comment:
This could be entitled to indicate that it covers typographic and trivial changes rather than exceptions.  I'm not sure what a good title would be however.

Reply:
"Minor edits" would do it for me...  and yeah I agree, that would be better than the general "Exceptions"

Reply:
Done